packages/components/src/local/route/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactNode } from 'react'
+import { LayerGroup, Polyline } from 'react-leaflet'
+
+import plannedRoutesFor from './helpers/planned-routes-for'
+import {RouteData} from './helpers/planned-routes-for'
```

Comment:
Lines 4 and 5 can be destructured, you can either do it as 
```suggestion
import plannedRoutesFor, { RouteData } from './helpers/planned-routes-for` 
```

Or you can alter the way the exports are done in `planned-routes-for` to allow them both to be pulled in with 

```suggestion
import { plannedRoutesFor, RouteData } from './helpers/planned-routes-for` 
```

The latter would be preferable but either way is acceptable.